src/app/code/community/Flagbit/Faq/controllers/Adminhtml/Faq/CategoryController.php

```diff
@@ -122,6 +122,33 @@ public function saveAction()
             
             // try to save it
             try {
+
+                if(isset($_FILES['icon']['name']) && (file_exists($_FILES['icon']['tmp_name']))) {
```

Comment:
could you pls use php-cs and check your files

Reply:
Sorry I don't know yet how to do this.

Reply:
Coding style checks should actually be part of the CI and break a build if the coding style doesn't match with the provided code. Is this about the default [Magento coding standard](https://github.com/magento/magento-coding-standard) or some else?